lesson4/public/indexmove.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title></title>
+</head>
+<body>
+  <div>
+  <button id = 'up'>UP</button>
+  <button id = 'down'>DOWN</button>
+  <button id = 'right'>RIGHT</button>
+  <button id = 'left'>LEFT</button>
+  <div id = 'pic'>
+    <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSnfpzoD-_HTIGH37ncMqaYiqjOI4MrXSPSTCyAHbJdSsc6O9vP">
+  </div>
+  </div>
+  <script>
+
+//Going up
+    const goUp = document.getElementById('up');
+    goUp.onclick = ()=>{
+      const t = 'up'
```

Comment:
please use better variable names throughout your file. Also, using a helper function here would help alot so you don't repeat your code.